packages/babel-types/src/traverse/traverseFast.ts

```diff
@@ -7,28 +7,39 @@ import type * as t from "../index.ts";
  */
 export default function traverseFast<Options = object>(
   node: t.Node | null | undefined,
-  enter: (node: t.Node, opts?: Options) => void,
+  enter: (node: t.Node, opts?: Options) => void | "skip" | "stop",
```

Comment:
Wdyt about returning `t.traferseFast.skip` and `t.traverseFast.stop` symbols instead of strings? We usually don't have this type of magic strings to control behavior in our public API.

Reply:
Strings seem to be a bit clearer in terms of type definition?

Reply:
I tried playing around with this, and it can be done: https://www.typescriptlang.org/play/?#code/MYewdgzgLgBA+hA1gSwA4C4YFczII5YCmMEAngLYBGIANjALwwDKF1NAFAJQDcAUAGY5gUZOBhQATgEMAboQkRCAMSnR2wSpi4MAfDBkhkAExgAfcaVSEQ-cdLkLlqqADokaTjADeAX16TZeUUVaDcUVAZ4d1Q+I0JgGikJYjApckIIVClgYgCHYOdvXhgSmEIAD1QQCVhQSFhozChLa1sEcL4-f3sgpzVtej0vYtKAelGYEEQRkuSoLAkwO0DHENdozs5eXnGYVTLK+KhCIwAaMokJaph2SixYKAALYnkriRh0iAgpAHNiZAgMDAIFgDlIMB+IBAZxgABUmCRHiAsDQTMhyKgrnIYMgoFs8r01uwBkMZjA5gsliwqLQuJttkA

However, the autocomplete experience is much better when using strings, so up to you. Feel free to merge this PR as-is if you still think strings are better.

Reply:
It looks like @JLHwung prefers symbols too, so I'll use that. :)

Reply:
It seems that TS allows us to use `typeof traverseFast.skip` directly.
![image](https://github.com/user-attachments/assets/1be20e27-7319-4d66-a5a6-6c91d91ce611)